device-types/Ivanti/PulseSecure-PSA300.yaml

```diff
@@ -0,0 +1,23 @@
+---
+manufacturer: Ivanti
+model: PulseSecure PSA300
+slug: ivanti-pulse-secure-psa300
+part_number: PSA300
+u_height: 1
+is_full_depth: false
+weight: 1.13
+weight_unit: kg
+airflow: front-to-rear
+comments: '[Pulse Secure Appliance Serie Data Sheet](https://secureaccessworks.com/datasheets/PulseSecure_OneSheet_PSAHardware.pdf)'
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 37
+interfaces:
+  - name: port1
```

Comment:
Are these actually called `port1` in the devices configuration? If so then you can mark this as resolved, otherwise please make sure this matches

Reply:
Your also right here. In GUI and XML Config the Interfaces are called "internal" and "external" on all appliances. 
Also the PSA7000 has 2 internal and external ports.
I fixed all enties.